client/src/client-only-routes/ShowCertification.js

```diff
@@ -311,7 +359,19 @@ const ShowCertification = props => {
           </footer>
         </Row>
       </Grid>
-      {signedInUserName === username ? shareCertBtns : ''}
+      {signedInUserName === username ? (
+        shareCertBtns
+      ) : (
+        <>
+          <Spacer size={2} />
+          <ShowProjectLinks
+            username={username}
+            user={user}
+            name={userFullName}
+            certName={certTitle}
+          />
+        </>
+      )}
```

Comment:
```suggestion
      {signedInUserName === username && shareCertBtns}
      <Spacer size={2} />
      <ShowProjectLinks
        username={username}
        user={user}
        name={userFullName}
        certName={certTitle}
      />
```
I think we always want the share buttons to be there.

Reply:
The share buttons, or the projects? Currently, I have not changed when the `shareCertBtns` are visible (only when the camper views their own cert), but I made it so the `ProjectLinks` are only visible to everyone else (not the camper themselves)

Reply:
Sorry, I misspoke - I meant the projects.  It seems a little weird (to me) that they are not visible when you're viewing your own cert. Let's leave this open until we've nailed down the technical issues.